v2/services/authorize/client.go

```diff
@@ -194,18 +194,18 @@ func (c *client) DialUsingCredentialsWithContext(ctx context.Context, sess *sess
 	return c.DialUsingCredentialsManager(ctx, cm, host, port, secretKey, opts...)
 }
 
-func (c *client) DialUsingCredentialsManager(ctx context.Context, cm *credentialsmanager.CredentialsManager, host, port, secretKey string, opts ...grpc.DialOption) error {
-	return c.withCredentialsManager(cm).
+func (c *client) DialUsingCredentialsManager(ctx context.Context, cf credentialsmanager.CredentialsFetcher, host, port, secretKey string, opts ...grpc.DialOption) error {
```

Comment:
Is the `*CredentialsManager` struct implementing the CredentialsFetcher interface?

Reply:
Yes the `CredentialsManager` does implement the `CredentialsFetcher` interface.